aries-site/src/pages/components/spinner.mdx

```diff
@@ -0,0 +1,69 @@
+import { Example } from '../../layouts';
+import {
+  AnnounceSpinnerExample,
+  SpinnerExample,
+  ListSpinnerExample,
+  ContentSpinnerExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/spinner/SpinnerExample.js"
+  docs="https://v2.grommet.io/spinner?theme=hpe"
+  figma="https://www.figma.com/file/feIeYk1zX36y9E7e7rprc2/HPE-Loading-Indicator-Component?node-id=0%3A1"
+>
+  <SpinnerExample />
+</Example>
+
+## Guidance
+
+Loading Spinners are used to notify the user that an action is being processed. 
+This reassures the user that something is occuring and will be provided shortly. 
+
+Progress circles help the user to show the progression of a system operation (downloadnig, uploading, processing, etc) visually. 
+### When to use Spinner
+
+Use a spinner when an action requires a short time to process (downloadnig, uploading, processing, etc).
+When loading more information into a table or list a spinner should be used.
+
+### Accessibility
+
+In order for a spinner to be accessible we need to provide a [message](https://v2.grommet.io/spinner#message) that will be announced for screen readers. 
```

Comment:
Is there any default message that Grommet provides? If no, then it feels like we should add a message on all of our examples to be accessible.

Reply:
There is no default message. I did provide a message for every example. I think it may be good if we have an option to do this in the theme in case someone forgets to provide the message prop. @ShimiSun would adding a default message in the theme be something you would be open to adding in grommet

Reply:
Chatted with Shimi on this. This message is most valuable when the message is specific to an application's use case, as opposed to something general. Given that, we don't need to add a default message in the theme at this time. Additionally, we can remove the messages from the examples that aren't the "announcement" one.

Reply:
thank you for clarifying